app/templates/profile/accounts.hbs

```diff
@@ -9,7 +9,26 @@
           <h1>{{userName}}</h1>
           <p>Repositories <strong>{{user.reposCount}}</strong></p>
         {{/link-to}}
-        <p class="profile-user-last">Token: <strong>{{auth.currentUser.token}}</strong></p>
+        <p class="profile-user-last">Token: 
+
+        {{#if isVisible}}
+          <strong>{{auth.currentUser.token}}</strong>
+        {{/if}}
+        <a {{action 'tokenVisibility'}} class="profile-token-toggle tooltip--profile {{if isVisible 'is-visible'}} dropdown">
+          <span class="tooltip-bubble">
+            {{#if isVisible}}hide token{{else}}show token{{/if}}
+          </span>
+          <span class="icon-eye">
+            <svg x="0px" y="0px" viewBox="2 -1 17 17" width="20" height="18" xml:space="preserve">
```

Comment:
I think that component will be easier to use, ie. you can put a template there. Helper is probably more lightweight, but I doubt it's worth to use it here (unless we need hundreds of these svgs or something like that ;) )